File: p2/python/p2.py
# INITIALIZE VARIABLES
num_prev = 1;       # first Fibonacci number
num_next = 2;       # second Fibonacci number
num_term = 10;      # total number of terms

# MAIN PROGRAM
if __name__ == "__main__":
    #print(str(num_prev) + ", " + str(num_next), end = '')
    summation = num_prev + num_next

    for _ in range(3, num_term + 1):
        temp = num_next
        num_next += num_prev
        num_prev = temp
        #print(", " + str(num_next), end = '')
        summation += num_next
    
    print(summation)

File: p2/c/p2.c
/* IMPORT LIBRARIES */
#include <stdio.h>

/* INITIALIZE VARIARBLES */
int num_prev = 1;	// 1st Fib. number
int num_next = 2;	// 2nd Fib. number
int num_lim = 4000000;	// Limit for max Fib. number

/* MAIN PROGRAM */
int main(void)
{
	int sum = 0;
	int temp;
	while (num_next <= num_lim)
	{
		if (num_next % 2 == 0) sum += num_next;
		temp =  num_next;
		num_next += num_prev;
		num_prev = temp;
	}

	printf("%d\n", sum);
	return 0;
}
